slurmci-test.toml

```
@@ -47,6 +47,9 @@ cpu_gpu = [
   { file = "test/Atmos/Parameterizations/Microphysics/KM_saturation_adjustment.jl", slurmargs = ["--ntasks=3"], args = [] },
   { file = "test/Atmos/Parameterizations/Microphysics/KM_warm_rain.jl", slurmargs = ["--ntasks=3"], args = [] },
   { file = "test/Atmos/Parameterizations/Microphysics/KM_ice.jl", slurmargs = ["--ntasks=3"], args = [] },
+  { file = "test/Land/Model/haverkamp_test.jl", slurmargs = ["--ntasks=1'"], args = [] },
```

```suggestion
  { file = "test/Land/Model/haverkamp_test.jl", slurmargs = ["--ntasks=1"], args = [] },
```
This is breaking slurmci. :(

---

oh nooo i had an extra apostrophe! and is it something to do with the spaces too?

---

thank you so much for catching this!

---

> oh nooo i had an extra apostrophe! and is it something to do with the spaces too?

Quite possibly, I'd try to stick very close to the existing pattern